src/utils/columnUtils.ts

```diff
@@ -109,7 +109,7 @@ export function canEdit<R, SR>(column: CalculatedColumn<R, SR>, row: R): boolean
   if (typeof column.editable === 'function') {
     return column.editable(row);
   }
-  return Boolean(column.editor || column.editable);
+  return column.editable || column.editor != null && column.editable !== false;
```

Comment:
editor | editable | result
-------|----------|-------
no | false | false
no | true | true => `!editor && editable`
yes | false | false
yes | true | true => `editor && editable`
no | nullish | false
yes | nullish | true => `editor && editable == null`

```
(!editor && editable || editor && editable)
  => editable
```
So you end up with
```js
editable || editor != null && editable != null
// or
editable || editor != null && editable !== false
```

Reply:
To be honest, I'm not sure about the `nullish` cases. I think to reduce the complexity for understanding it would be nice to make sure the editor doesn't open when the `editable` is not set to any value. It means less flexibility, but helpful for consumer dev debugging.

Reply:
Adding `editable: true` for all your columns with editors is cumbersome and unintuitive IMO.

Reply:
It is cumbersome but more precise.